data/helpers.d/backend

```diff
@@ -213,3 +213,209 @@ ynh_remove_fpm_config () {
 	ynh_secure_remove "$fpm_config_dir/conf.d/20-$app.ini" 2>&1
 	sudo systemctl reload $fpm_service
 }
+
+# NODEJS HELPERS
+
+n_install_dir="/opt/node_n"
+node_version_path="/opt/node_n/n/versions/node"
+# N_PREFIX is the directory of n, it needs to be loaded as a environment variable.
+export N_PREFIX="$n_install_dir"
+
+# Install Node version management
+#
+# [internal]
+#
+# usage: ynh_install_n
+ynh_install_n () {
+	echo "Installation of N - Node.js version management" >&2
+	# Build an app.src for n
+	mkdir -p "../conf"
+	echo "SOURCE_URL=https://github.com/tj/n/archive/v2.1.7.tar.gz
```

Comment:
When do you think we will have to update that reference version?

Reply:
Well, if we need to maintain these helpers, do we only need to rely on CI results on apps using them to know if there something to change here (link broken, security fix, etc.)?

Reply:
I'm not so sure to understand the point.
We have here the last available release, there's no other release since april 2017.
What else do you want we do ?

Reply:
I'm asking the question for the future maintenance.
Do we need to follow tightly new n version releases?

Reply:
I don't think so unless there's a big modification of how node can be handle.
But since we used here a specific version, I don't think it would be any problem if there's any new version of n.

Except a big modification of node, or a big security issue about n, I think if it would be any upgrade of n, we can take our time to check it and implement it.